### PR TITLE
[imp] Highlights the tasks with date_end > date_deadline

### PR DESCRIPTION
**Pull request by karlis-dreizis**
Helps the user see, straight from the timeline view, whether his/hers
set end_date meets the set date_deadline.

Depends on OCA/web#1206

### CODE REVIEW

File: project_timeline/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Project timeline",
     "summary": "Timeline view for projects",
-    "version": "11.0.1.1.0",
+    "version": "11.1.0.0.0",
```
Comment (nikul-serpentcs) [inline project_timeline/__manifest__.py]: @InfernalLV `11.0.1.0.0`

File: project_timeline/__manifest__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
     "category": "Project Management",
     "website": "https://www.tecnativa.com",
```
Comment (nikul-serpentcs) [inline project_timeline/__manifest__.py:10]: @InfernalLV Should be `https://github.com/OCA/project`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

File: project_timeline/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply project_timeline/__manifest__.py]: It should `11.0.1.2.0`

Comment (karlis-dreizis) [reply project_timeline/__manifest__.py]: I see what I got wrong. But shouldn't it then be `11.0.2.0.0`? This pull request addresses a breaking change in OCA/web#1206. Doesn't that mean that this pull request introduces a breaking change as a consequence of the first one? Which Is why I went for the bump on the highest digit.

Comment (pedrobaeza) [reply project_timeline/__manifest__.py]: Yes, 11.0.2.0.0 makes sense.

Comment (karlis-dreizis) [reply project_timeline/__manifest__.py]: Version bumped

**Comment by karlis-dreizis** [comment]
Am I supposed to address the ci/runbot error? 
Seems this could be what causes the failure `fatal: Couldn't find remote ref refs/merge-requests/480/head`

**Comment by karlis-dreizis** [comment]
I inspected some other PR's and took a look at runbot repo. Based on https://github.com/odoo/runbot/blob/master/runbot/models/build.py#L22 and its usage I'd venture to say that ci/runbot fails because of following line from the log
`2019-03-08 12:02:02,204 193 WARNING openerp_test odoo.modules.registry: Models have no table: report.project.task.user.`

Again, how do I address this? If I do at all? The warning is unrelated to my PR. The report.project.task.user model is a sql view backed report from the `project` module.

I'd like to get this merged as soon as reasonably possible. So any guidance would be greatly appreciated.

Thanks!

**Comment by pedrobaeza** [comment]
Indeed, this is not your fault, so unrelated to your PR and you can leave it. Let's see if that's fixable in OCA itself or in Odoo.

**Pull request closed without merging**